laevatein/build.gradle

```diff
@@ -17,15 +17,13 @@ android {
     testOptions.unitTests.includeAndroidResources true
 }
 
-ext.supportLibraryVersion = '28.0.0'
-
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "com.android.support:support-v4:${supportLibraryVersion}"
-    implementation "com.android.support:appcompat-v7:${supportLibraryVersion}"
-    implementation "com.android.support:support-annotations:${supportLibraryVersion}"
-    implementation "com.android.support:design:${supportLibraryVersion}"
-    implementation "com.android.support:exifinterface:${supportLibraryVersion}"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'androidx.appcompat:appcompat:1.1.0-alpha02'
```

Comment:
Do you have a reason to specify alpha version?
I think that to use the stable version (1.0.2).
see: https://mvnrepository.com/artifact/androidx.appcompat/appcompat

Reply:
Ah, thank you good comment 😘 
I fix to use stable version.

> Do you have a reason to specify alpha version?

Auto specified alpha versionI , when used `Refactor > Migrate to AndroidX...` on Android Studio.
I was not aware of this, sorry.

Reply:
@mrtry I fixed this on https://github.com/nohana/Laevatein/pull/86/commits/5a05f6808f921511739954234d1cbd0237ab6e55 .
Please check again, thank you 😄